src/redux/store.jsx

```diff
@@ -0,0 +1,12 @@
+import { configureStore } from '@reduxjs/toolkit';
+import counterReducer from './counter/counterSlice';
+import booksReducer from './books/bookSlice';
+import catReducer from './categories/catSlice';
+
+export const store = configureStore({
+  reducer: {
+    counter: counterReducer,
+    books: booksReducer,
+    categories: catReducer,
+  },
+});
```

Comment:
- [ ] Please make sure to use the `.js` file extension instead of the `.jsx` extension, You'll be only using javascript code in redux files. (This is applicable to books and categories slice too).